rfb/rfbconfig.h.cmakein

```diff
@@ -153,9 +153,9 @@
 /* Define to the version of this package. */
 #define LIBVNCSERVER_PACKAGE_VERSION  "@LibVNCServer_VERSION@"
 #define LIBVNCSERVER_VERSION "@LibVNCServer_VERSION@"
-#define LIBVNCSERVER_VERSION_MAJOR "@LibVNCServer_VERSION_MAJOR@"
-#define LIBVNCSERVER_VERSION_MINOR "@LibVNCServer_VERSION_MINOR@"
-#define LIBVNCSERVER_VERSION_PATCHLEVEL "@LibVNCServer_VERSION_PATCH@"
+#define LIBVNCSERVER_VERSION_MAJOR @LibVNCServer_VERSION_MAJOR@
```

Comment:
While this is correct and understandable, it changes existing API that users might rely on in their builds. So I fear this could only go in once a new major version is done.